connector/core/core-service/src/test/java/org/oaebudt/edc/core/SqlParticipantGroupStoreTest.java

```diff
@@ -0,0 +1,111 @@
+package org.oaebudt.edc.core;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.eclipse.edc.junit.annotations.PostgresqlIntegrationTest;
+import org.eclipse.edc.sql.QueryExecutor;
+import org.eclipse.edc.sql.testfixtures.PostgresqlStoreSetupExtension;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.oaebudt.edc.core.store.SqlParticipantGroupStore;
+import org.oaebudt.edc.spi.store.ParticipantGroup;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@PostgresqlIntegrationTest
+@ExtendWith(PostgresqlStoreSetupExtension.class)
+public class SqlParticipantGroupStoreTest {
+
+    private SqlParticipantGroupStore store;
+
+    @BeforeEach
+    void setUp(PostgresqlStoreSetupExtension extension, QueryExecutor queryExecutor) throws IOException {
+        var inputStream = getClass().getClassLoader().getResourceAsStream("schema.sql");
+        assert inputStream != null;
+        var schema = new String(inputStream.readAllBytes());
+        extension.runQuery(schema);
+
+        var dataSourceRegistry = extension.getDataSourceRegistry();
+        var datasourceName = extension.getDatasourceName();
+        this.store = new SqlParticipantGroupStore(
+                dataSourceRegistry,
+                datasourceName,
+                extension.getTransactionContext(),
+                new ObjectMapper(),
+                queryExecutor
+        );
+    }
+
+    @AfterEach
+    void tearDown(PostgresqlStoreSetupExtension extension) throws IOException {
+        var inputStream = getClass().getClassLoader().getResourceAsStream("schema-teardown.sql");
```

Comment:
suggestion: unless it's really needed generally I I would not create and destroy everything on every tests, but I'd make the tests independent between themselvs, like by using randomly generated ids for the groups and less strict assertions.
This way tests could be parallelized if needed.

Reply:
Done!!